src/components/Icons/IconClose.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
```

Comment:
Can we move this Icon to the same folder as `Modal`? It's not used anywhere else (yet, at least), so best to keep it with `Modal`.